ocp_resources/datavolume.py

```diff
@@ -281,7 +284,7 @@ def wait_for_dv_success(
         Args:
             timeout (int):  Time to wait for the DataVolume to succeed.
             failure_timeout (int): Time to wait for the DataVolume to have not Pending/None status
-            dv_garbage_collection_enabled (bool, default: False): if True, expect that DV will disappear after success
+            dv_garbage_collection_enabled (bool, default: False): garbage collector is removed in version v1.62 and going to be deprecated
```

Comment:
`v1.62` refers to what version? CDI/Kubevirt etc? Is it going to be removed or already removed?